Submodels/file_handling.py

```
@@ -0,0 +1,689 @@
+"""
+
+import numpy as np
+from scipy import interpolate
+from Parameterizations import AirfoilParameterization
+
+class fileHandling:
+    """
+    This class contains all methods needed to generate the required input files walls.xxx and tflow.xxx for an MTFLOW analysis. 
+    """
+
+    def __init__(self, *args) -> None:
```

Ruff (0.8.2)

Missing type annotation for `*args`

(ANN002)